default.json

```
@@ -16,10 +16,11 @@
   "commitMessagePrefix": "[BUMP]",
   "rebaseWhen": "never",
   "schedule": "every 1 hour every weekday",
+  "rangeStrategy": "pin",
   "packageRules": [
     {
       "matchUpdateTypes": ["patch", "minor"],
```

ça me va d'avoir des versions pas pin comme actuellement, et d'avoir des updates de lockFile, et pour éviter le changement massif de package.json pour passer en pin je propose plutôt d'essayer ça : 
```suggestion
      "matchUpdateTypes": ["lockFileMaintenance"],
```

---

Le [lockfile maintenance recréé le package-lock.json](https://docs.renovatebot.com/configuration-options/#lockfilemaintenance) à chaque maintenance et se base sur la config settée dans les package.json (`^`) si je comprend bien.

Par contre on a un conflit entre la fréquence de run de la maintenance `:maintainLockFilesMonthly` et certaines mises à jour qu'on aimerait avoir plus rapidement (voir le preset [`group-global-dependencies`](https://github.com/1024pix/renovate-config/blob/main/presets/group-global-dependencies.json)). Comment est-ce qu'on pourrait mélanger les 2 sans modifier le package.json ?

---

Je crois que je commence à comprendre le fonctionnement... En fait je pense que ça serait utile de séparer à minima nos dépendances internes dans leurs propres PRs plutôt que dans les lockfile maintenance. Ça permettrait d'avoir un meilleur historique de ces dépendances un peu particulières et de leur donner leur propre cycle de livraison : le plus tôt possible si la CI passe (pas besoin d'attendre un mois dans le pire cas).

Par contre je ne sais pas quelle est la meilleure solution pour réaliser ça : 
- Pin les dépendances concernées dans les package.json ?
- Changer la `rangeStrategy` à `pin` pour les `packageRules` concernés ?

Du coup pour moi l'`automerge` pour le `lockFileMaintenance` que tu proposes est utile (c'est cool si on l'a d'ici ce vendredi pour confirmer que ça automerge (1er de chaque mois)).

Mais j'aimerai bien aussi séparer les PRs de `@1024pix` comme dit plus haut.